src/index.js

```diff
@@ -1,152 +1,29 @@
 'use strict';
 
-let sql = null;
-let template = null;
+import Exporter from './exporter';
+export {SEPARATOR} from './exporter';
 
-if (process.env.APP_ENV === 'browser') {
+const isBrowser = process.env.APP_ENV === 'browser';
```

Comment:
Not a good idea, this statement is supposed to be replaced by `webpack` on bundling step and it's better to point it out explicitly in every `if`

```js
if (process.env.APP_ENV === 'browser') {
```